app/pages/reset-password/submit-email-form.spec.js

```diff
@@ -42,16 +42,14 @@ describe('SubmitEmailForm', function () {
   it('conditionally shows and hides an email success icon and message', function () {
     wrapper.setProps({ emailSuccess: true });
     assert.equal(wrapper.find('i.form-help.success').length, 1);
-    assert.equal(wrapper.find('Translate').last().prop('content'), 'resetPassword.emailSuccess');
+    assert.equal(wrapper.find('#reset-status').find('Translate').prop('content'), 'resetPassword.emailSuccess');
```

Comment:
`wrapper.find(Translate[content="resetPassword.emailSuccess"])` should find the success message, without having to use an ID. You can then test whether it's present, as expected.

Reply:
Nice - i'll update with the more succinct selectors, thanks.